src/shared/lib/StoreUtils.spec.ts

```diff
@@ -108,6 +109,27 @@ describe('StoreUtils', () => {
             }),
         ];
 
+        mutationDataWithNewEvent =  {
```

Comment:
Should we name this variable as `uncalledMutationDataWithNewEvent` or something like that. This is uncalled mutation data, right?